src/main/java/string/Exercise04.java

```diff
@@ -0,0 +1,23 @@
+package string;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Exercise04 {
+    public String cutString(final String str){
+        HashMap<String,Integer> map=new HashMap<>();
+        StringBuilder result=new StringBuilder();
+        for(Character c:str.toCharArray()){
+            map.put(c.toString(),map.getOrDefault(c.toString(),0)+1);
+        }
+        for(Map.Entry<String,Integer> entry:map.entrySet()){
+            if(entry.getValue()>1)
```

Comment:
Khi xuống dòng thì bắt buộc theo format


```suggestion
            if(entry.getValue() > 1) {
             ....
            }
```